test/new-e2e/pkg/utils/e2e/client/ecs/ecs.go

```diff
@@ -0,0 +1,101 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package ecs
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	awsConfig "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ecs"
+	"github.com/cenkalti/backoff/v4"
+)
+
+// Client is a client for ECS
+type Client struct {
+	ecs.Client
+	clusterName string
+}
+
+// NewClient creates a new ECS client
+func NewClient(clusterName string) (*Client, error) {
+	ctx := context.Background()
+	cfg, err := awsConfig.LoadDefaultConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	return &Client{Client: *ecs.NewFromConfig(cfg), clusterName: clusterName}, err
+}
+
+// ExecCommand executes a command in a container in a task in an ECS cluster.
+// It accepts either the task ARN or the task ID.
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**